test/resource_cluster_test.go

```diff
@@ -45,6 +45,103 @@ func TestAccCluster(t *testing.T) {
 	})
 }
 
+func TestKafkaConnectClusterCreateInstaclustrAWS(t *testing.T) {
+        if v := os.Getenv("IC_TEST_KAFKA_CONNECT"); v == "" {
+                t.Skip("Skipping TestKafkaConnectClusterCreateInstaclustrAWS because IC_TEST_KAFKA_CONNECT is not set")
```

Comment:
This is a really good idea, thanks for implementing it. Could you please modify the README by separating the kafka connect specific variables and making it clear those variables are optional?

Reply:
Acknowledged. Updated the README.md, what do you think?